Gemfile

```diff
@@ -12,6 +12,7 @@ gem "lita-office-automation", github: "platanus/lita-office-automation"
 gem "lita-pull-requests", github: "platanus/lita-pull-requests"
 gem "lita_time_tracker", github: "platanus/lita_time_tracker"
 gem "lita-trainer-bot", github: "platanus/lita-trainer-bot"
+gem "lita-slack-actions", github: "platanus/lita-slack-actions"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem lita-slack-actions should appear before lita-trainer-bot.